README.adoc

```diff
@@ -195,6 +195,16 @@ using for that specific processor.
 metanorma version --type iso
 ----
 
+=== Add new metanorma template (`metanorma template add`)
```

Comment:
We want to clarify the terminology:

* A Metanorma "template repository" contains multiple Metanorma "templates". A template repository is a Git Repo (e.g. https://github.com/metanorma/mn-templates-ogc/).

* A Metanorma "template" is a directory within a Metanorma "template repository". 

The code right now adds "template repositories", not "templates". However, it may be useful to add a single "template" for use.

When we list templates, we want to know:

```
for every template-repository configured
  display name and URL of the template-repository
end
```

When we list templates, we want to know:

```
for every template-repository configured
  for every template in the template-repository
    display name of the template
  end
end
```

Reply:
Cool, so as a first step can we say we allow the user to add template repository now and later if that becomes important then we can consider having an option for each individual template.

Btw, apart from documentation, do want me to change anything how we want to call the interface?

Reply:
I think it's fine to just support template-repository for now. This means we should rename the command `metanorma template ...` to `metanorma template-repo ...` or `metanorma template repo ...`?